test/factories/locations.rb

```diff
@@ -0,0 +1,11 @@
+FactoryGirl.define do
+  factory :location do
+    va_accepted false
+    address1 "MyString"
```

Comment:
Here too, removing any `MyString`.

Reply:
@hpjaj if I use Faker I'd have to add some more code to the tests, do you want me to user faker on all of the attributes?

Reply:
Adding more code to the tests is fine.  The more representative our tests are of the real world, the better.

Yes, please use faker in situations where uniqueness is valuable, and more representative of the real world.  

You can still choose specific attributes in the tests if needs be, i.e.

```ruby
let(:location) { create :location, address1: "1515 Main St.", .. }
```

Reply:
Ok let me do all of this and ping you once I've completed. Thanks for your help~!